go.mod

```diff
@@ -123,13 +126,15 @@ require (
 	google.golang.org/grpc v1.23.0
 	gopkg.in/gorp.v1 v1.7.2 // indirect
 	gopkg.in/yaml.v2 v2.2.2
+	helm.sh/helm v2.14.3+incompatible
```

Comment:
looks like the go.mod file sneaked in a few accidental adds. Would you mind reverting these?